src/main/java/randoop/generation/OrienteeringSelection.java

```diff
@@ -168,12 +167,8 @@ private double computeTotalWeightForCandidates(SimpleList<Sequence> candidates)
       SequenceDetails details = sequenceDetailsMap.get(candidate);
       if (details == null) {
         // This might be a literal that was created by ComponentManager.getSequencesForType().
-        throw new Error(
-            String.format(
-                "candidate is not in sequenceDetailsMap.%n"
-                    + "candidate [%s]:%n%s%nEnd of candidate.%n"
-                    + "sequenceDetailsMap:%n%s%nEnd of sequenceDetailsMap%n",
-                candidate.getClass(), candidate, CollectionsPlume.mapToString(sequenceDetailsMap)));
+        createdExecutableSequence(new ExecutableSequence(candidate));
```

Comment:
That would be ideal. However, I think all available sequences are added when sequenceDetailsMap is created already. I am not sure where, the problematic sequemces are being introduced. If we can figure that out it would be simple to add them into the map.